server/src/dbschema.js

```diff
@@ -6,7 +6,7 @@ const mozlog = require("./logging").mozlog("dbschema");
 
 // When updating the database, please also run ./bin/dumpschema --record
 // This updates schema.sql with the latest full database schema
-const MAX_DB_LEVEL = exports.MAX_DB_LEVEL = 18;
+const MAX_DB_LEVEL = exports.MAX_DB_LEVEL = 19;
```

Comment:
Huh, oddly the database schema update has already been deployed (well, not oddly, but purposely) – but oddly it's showing up in this PR, since it's otherwise landed.